programs/protocol-contracts-solana/src/lib.rs

```diff
@@ -351,6 +360,11 @@ pub struct DepositSplToken<'info> {
     #[account(seeds = [b"meta"], bump)]
     pub pda: Account<'info, Pda>,
 
+    #[account(seeds=[b"whitelist", mint_account.key().as_ref()], bump)]
+    pub whitelist_entry: Account<'info, WhitelistEntry>, // attach whitelist entry to show the mint_account is whitelisted
+
+    pub mint_account: Account<'info, Mint>,
```

Comment:
What does mint mean? 

There should be no minting involved since we lock/unlock SPL tokens?

Reply:
> What does mint mean?
> 
> There should be no minting involved since we lock/unlock SPL tokens?

mint (short for mint account) is the Solana equivalent for ERC20 contract address. 
This is Solana jargon.